### PR TITLE
Feature/3140 deploy command and deploy schema command should not redeploy schemas

### PR DESCRIPTION
**Pull request by MHove89**
Avoiding deployments of schemas that are already deployed on the environment.

### CODE REVIEW

**Review by jthind — APPROVED** [review]

File: src/Enterspeed.Cli/Commands/Deploy/DeployCommand.cs [hunk 1]
```diff
@@ -130,7 +131,23 @@ private async Task<IReadOnlyList<GetMappingSchemaResponse>> FetchSchemas(List<De
...
+                    exstingDeployedSchemaForEnvironment.Version != schema.Version.Id.Version)
+                {
+                    schemas.Add(schema);
+                }
+                // If existing deployed schema for environment is null, then there are no deployed schema of this type
+                // on the environment. In this case we add the schema to list of schemas to deploy.
+                else if (exstingDeployedSchemaForEnvironment == null)
+                {
+                    schemas.Add(schema);
+                }
```
[the remark this answers is not in the document]
Comment (MHove89) [reply src/Enterspeed.Cli/Commands/Deploy/DeployCommand.cs:150]: You are right on both comments. :) Forget the force argument. - I will get it fixed :)

[the remark this answers is not in the document]
Comment (jesperweber) [reply src/Enterspeed.Cli/Commands/Deploy/DeployCommand.cs:150]: Super :-) The force argument, should only be on `DeploySchemaCommand` and not on `DeployCommand`

[the remark this answers is not in the document]
Comment (MHove89) [reply src/Enterspeed.Cli/Commands/Deploy/DeployCommand.cs]: Changes has been implemented @jesperweber

**Review by jesperweber — APPROVED** [review]
Nice, LGTM

**Pull request merged**